bucket/windiff.json

```
@@ -0,0 +1,17 @@
+{
+    "version": "2.1.0",
+    "description": "The omnipotent, omniscient, omnipresent diffing program",
+    "homepage": "https://en.wikipedia.org/wiki/WinDiff",
+    "license": "UNLICENSED",
+    "architecture": {
```

32bit programs can run on 64bit, so the architecture tag can be shortened to:
```suggestion
        "url": "https://github.com/kitamstudios/WinDiff/releases/download/refs%2Fheads%2Fmaster/WinDiff.zip",
        "hash": "e25ccdc2fef77eb001fc3a5d755d448a301338d7e4feb2cf365bcb3d34200931",
```
Also can you add an `autoupdate`?

---

thanks for you review :)

i dont have the code building yet. so it is just the checked in version. i was planning to make the auto update once that work is done.

let me know if that is not ok with you.